jung-visualization/src/main/java/edu/uci/ics/jung/visualization/BasicVisualizationServer.java

```diff
@@ -224,12 +228,19 @@ public void setRenderer(Renderer<V, E> r) {
     return renderer;
   }
 
-  public void setGraphLayout(Layout<V> layout) {
+  public void setLayoutMediator(LayoutMediator<V, E> layoutMediator, Dimension d) {
+    model.setLayoutMediator(layoutMediator, d);
+  }
+
+  public void setLayoutMediator(LayoutMediator<V, E> layoutMediator) {
+    if (log.isDebugEnabled()) {
+      log.debug("setLayoutMediator to " + layoutMediator);
+    }
```

Comment:
It's not really clear to me that a JUNG user would find this logging useful. After all, the typical developer has access to a debugger for cases like this, and if JUNG ends up used in a command-line application, we would clutter the output log, annoying users (who would experience it firsthand) and developers (who would have to find a way to control or disable it) alike.